share/spice/bible/bible.js

```diff
@@ -14,6 +14,7 @@
             id: 'bible',
             name: 'Answer',
             data: result,
+            signal: 'high',
```

Comment:
Ah, OK. So for IAs that aren't appearing as the default tab but should be, we should set it internally instead?